src/components/data-table/_data-table-action.scss

```diff
@@ -242,14 +252,16 @@
     top: 0;
     left: 0;
     align-items: center;
-    opacity: 0;
     padding-left: $spacing-lg;
     padding-right: $spacing-lg;
     width: 100%;
     height: 100%;
-    z-index: z('header');
-    background-color: transparent;
-    transition: opacity $transition--base $carbon--standard-easing, background-color $transition--base $carbon--standard-easing;
+    will-change: transform;
+    background-color: $brand-01;
```

Comment:
should we use a different token here? I thought `$brand-01` was deprecated in v10? https://github.com/IBM/carbon-elements/blob/master/docs/migration/10.x-color.md

Reply:
That might have been my fault if I left it in there. I thought I updated all the tokens.

Reply:
Good catch 👍

Reply:
yeah `$brand-01` should be replaced with `$interactive-01`